tensorflow-framework/src/main/java/org/tensorflow/framework/initializers/TruncatedNormal.java

```diff
@@ -31,13 +30,12 @@
  *     TruncatedNormal&lt;TFloat32, TFloat32&gt; initializer =
  *              new org.tensorflow.framework.initializers.TruncatedNormal&lt;&gt;(tf, seed);
  *     Operand&lt;TFloat32&gt; values =
- *              initializer.call(tf.constant(Shape.of(2,2)), TFloat32.DTYPE);
+ *              initializer.call(tf.constant(Shape.of(2,2)), TFloat32.class);
  * </pre>
  *
  * @param <T> The TType for the call operation
- * @param <U> The TNumber for the call operation
  */
-public class TruncatedNormal<T extends TType, U extends TNumber> extends BaseInitializer<T> {
+public class TruncatedNormal<T extends TNumber> extends BaseInitializer<T> {
```

Comment:
Shouldn't this be `TFloating`?

Reply:
Check was only one numbers, [see this comment](https://github.com/tensorflow/java/pull/174#discussion_r547489320)

Reply:
TrunctedNormal, RandomNormal should be TFloating. 
RandomUniform can be TInteger and TFloating, so that should be TNumber.

Reply:
Ok so I'll do the corrections for `TruncatedNormal` and `RandomNormal`